pyaedt/application/Analysis.py

```diff
@@ -2025,7 +1973,8 @@ def _export_touchstone(
 
     @pyaedt_function_handler()
     def value_with_units(self, value, units=None):
-        """Combine a number and a string containing the unit in a single string e.g. "1.2mm".
+        """Combine a number and a string containing the modeler length unit in a single
+        string e.g. "1.2mm".
```

Comment:
```suggestion
        string. For example, "1.2mm".
```
Don't use Latin phrases.